### PR TITLE
Technical/refactoring

### PR DESCRIPTION
**Pull request by hasmikharutyunyan**

### CODE REVIEW

File: HappyTravel.Nakijin.Api/Services/Workers/AccommodationsMapping/IAccommodationsMapperDataRetrieveService.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        Task<Dictionary<string, int>> GetLocalitiesByCountry(int countryId);
+
+
+        Task<Dictionary<(int LocalityId, string LocalityZoneName), int>> GetLocalityZonesByCountry(
+            int countryId);
+
+
+        Task<Dictionary<int, (int Id, HashSet<int> MappedHtIds)>> GetHtAccommodationMappings();
+
+        Task<DateTime> GetLastMappingDate(Suppliers supplier, CancellationToken cancellationToken);
```
Comment (mingazhev) [inline HappyTravel.Nakijin.Api/Services/Workers/AccommodationsMapping/IAccommodationsMapperDataRetrieveService.cs:39]: One line between methods in one case and two lines in another

**Review by mingazhev — APPROVED** [review]
With comments

File: HappyTravel.Nakijin.Api/Controllers/AccommodationsManagementController.cs [hunk 2]
```diff
@@ -136,7 +138,7 @@ public IActionResult MapAccommodationsFully([FromBody] List<Suppliers> suppliers
                 {
                     try
                     {
-                        var mapper = scope.ServiceProvider.GetRequiredService<IAccommodationMapper>();
+                        var mapper = scope.ServiceProvider.GetRequiredService<IAccommodationsMapper>();
```
Comment (kirillta) [inline HappyTravel.Nakijin.Api/Controllers/AccommodationsManagementController.cs:141]: `IAccommodationMapper` is the right name

File: HappyTravel.Nakijin.Api/Controllers/LocationsManagementController.cs [hunk 3]
```diff
@@ -43,7 +44,7 @@ public IActionResult MapLocations([FromBody] List<Suppliers> suppliers)
                 {
                     try
                     {
-                        var locationService = scope.ServiceProvider.GetRequiredService<ILocationMapper>();
+                        var locationService = scope.ServiceProvider.GetRequiredService<ILocationsMapper>();
```
Comment (kirillta) [inline HappyTravel.Nakijin.Api/Controllers/LocationsManagementController.cs:47]: And `ILocationMapper` here as well

File: HappyTravel.Nakijin.Api/Infrastructure/Logging/LoggerExtensions.g.cs [hunk 4]
```diff
@@ -49,43 +49,43 @@ static LoggerExtensions()
...
             
             MappingLocalityZonesOfSpecifiedCountryStartOccured = LoggerMessage.Define<string>(LogLevel.Information,
                 new EventId(90111, "MappingLocalityZonesOfSpecifiedCountryStart"),
-                $"INFORMATION | LocationMapper: {{message}}");
+                $"INFORMATION | LocalityZonesMapper: {{message}}");
             
             MappingLocalityZonesOfSpecifiedCountryFinishOccured = LoggerMessage.Define<string>(LogLevel.Information,
                 new EventId(90112, "MappingLocalityZonesOfSpecifiedCountryFinish"),
-                $"INFORMATION | LocationMapper: {{message}}");
+                $"INFORMATION | LocalityZonesMapper: {{message}}");
```
Comment (kirillta) [inline HappyTravel.Nakijin.Api/Infrastructure/Logging/LoggerExtensions.g.cs:88]: Additional subject must be in a singular form generally

File: HappyTravel.Nakijin.Api/Services/Workers/MultilingualDataHelper.cs [hunk 5]
```diff
@@ -49,49 +49,59 @@ public MultilingualDataHelper(ILocationNameNormalizer locationNameNormalizer)
...
+                normalizedLocalityNames.TrySetValue(name.languageCode,
+                    _locationNameNormalizer.GetNormalizedLocalityName(defaultCountry, name.value));
+
+            return normalizedLocalityNames;
+        }
+
+
+        public MultiLanguage<string>? NormalizeMultilingualNames(in MultiLanguage<string>? name)
+        {
+            if (name == null)
```
Comment (kirillta) [inline HappyTravel.Nakijin.Api/Services/Workers/MultilingualDataHelper.cs:92]: use `is` whenever possible

File: HappyTravel.Nakijin.Api/Services/Workers/AccommodationsMapping/IAccommodationsMapperDataRetrieveService.cs [hunk 6]
```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using HappyTravel.Nakijin.Api.Models.Mappers;
+using HappyTravel.Nakijin.Data.Models;
+using NetTopologySuite.Index.Strtree;
+using Contracts = HappyTravel.EdoContracts.Accommodations;
+
```
Comment (oleg1std) [inline HappyTravel.Nakijin.Api/Services/Workers/AccommodationsMapping/IAccommodationsMapperDataRetrieveService.cs:9]: extra line

File: HappyTravel.Nakijin.Api/Infrastructure/Logging/LoggerExtensions.g.cs (hunk 4 above)
Comment (hasmikharutyunyan) [reply HappyTravel.Nakijin.Api/Infrastructure/Logging/LoggerExtensions.g.cs:88]: Do You mean  `MappingLocalityZonesOfSpecifiedCountryFinish` name in singular form, or `LocalityZonesMapper` ?

Comment (kirillta) [reply HappyTravel.Nakijin.Api/Infrastructure/Logging/LoggerExtensions.g.cs:88]: Both

**Review by kirillta — CHANGES_REQUESTED** [review]
All names must be fixed

**Review by kirillta — APPROVED** [review]

**Pull request merged**